src/app.ts

```diff
@@ -58,7 +58,7 @@ app.post('/checkout-session', async (req, res) => {
     params = CheckoutParamsSchema.parse(req.body);
   }
   catch (error) {
-    res.status(400).send(error);
+    return res.status(400).send(error);
```

Comment:
I thin the is not neccesary to return a value for a express endpoint response. Just by setting the response status and calling send the error will be sent out of the api.